tensorflow/lite/micro/benchmarks/README.md

```diff
@@ -64,3 +64,28 @@ make -f tensorflow/lite/micro/tools/make/Makefile TARGET=sparkfun_edge person_de
 
 Refer to flashing instructions in the [Person Detection Example](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/micro/examples/person_detection/README.md#running-on-sparkfun-edge).
 
+
+## Run on FVP based on Arm Corstone-300 software.
+For more info about the Corstone-300 software see: tensorflow/lite/micro/cortex_m_corstone_300/README.md.
```

Comment:
the formatting is a bit odd in markdown.

Can you make all the links clickable (relative paths to the README.md would be fine) and also add in a few newlines to properly demarcate the paragraphs in the rendered markdown.

Reply:
Yes will do